tox.ini

```diff
@@ -14,7 +14,7 @@ commands = pytest {posargs} tests/
 
 [flake8]
 extend-ignore = E203
-extend-exclude = .tox,./env
+extend-exclude = .tox,./env,./gspread/__init__.py
```

Comment:
why do we skip `__init__.py` from flake8 ? :thinking:

Reply:
Flit takes the `pyproject.toml`'s `project.description` from the docstring of the `__init__.py` file of the module. Thus, I changed this in https://github.com/burnash/gspread/commit/35639014f87c7b0977f1521d06a848cfb46fcebc.

You will see that at the top of the file was `#flake8: noqa` which skips the file when using flake8.

https://github.com/burnash/gspread/blob/97ab607bc3212e47157e706129743c586d342c95/gspread/__init__.py#L1

Thus, my change is just to move where this ignore is specified. The file was already ignored.

However, as for *why*, it is because `flake8` complains about unused imports in the `__init__.py` file:

```bash
.\dist\gspread\gspread\__init__.py:7:1: F401 '.auth.authorize' imported but unused
from .auth import (
^
```

etc., as many things are imported (so that users can do `import * from gspread`, and other reasons.

https://github.com/burnash/gspread/blob/a4789287ed7bbb0db5c42d0ba3c69c735fe08211/gspread/__init__.py#L7-L25

Reply:
Oh ok that makes sense then to ignore it with the rest. 

It's better this way.

I see for the import. Let's skip it.

It's all good to me.